app/services/acuant/liveness_pass_through.rb

```diff
@@ -0,0 +1,62 @@
+module Acuant
```

Comment:
Just appeasing code here.  If all goes well with the mainline doc auth and selfie checking this code and the scan id controller code will be deprecated.